### PR TITLE
Merge incubation branch commits

### PR DESCRIPTION
**Pull request by VaishnaviHire**
## Merge criteria

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: .github/workflows/unit-tests.yaml [hunk 1]
```diff
@@ -23,6 +23,10 @@ jobs:
         run: make unit-test
 
       - name: Upload results to Codecov
+<<<<<<< HEAD
```
Comment (zdtsw) [inline .github/workflows/unit-tests.yaml:26]: need cleanup

File: apis/infrastructure/v1/nim_types.go [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package v1
```
Comment (zdtsw) [inline apis/infrastructure/v1/nim_types.go:1]: this should have been moved to kserver_type.go

Comment (zdtsw) [reply apis/infrastructure/v1/nim_types.go:1]: probably you need rebase your "for" branch first

Comment (VaishnaviHire) [reply apis/infrastructure/v1/nim_types.go:1]: I think I just missed removing this file

**Pull request merged**